woocommerce/woocommerce-gutenberg-products-block
# Allow HTML in the All Products Block Product Titles

Uses `dangerouslySetInnerHTML` to allow HTML in product titles. Titles coming from the StoreAPI are ran through `wp_kses_post`. Note that core also allows HTML in product titles and does not face the same problem.

Fixes #1685

### Screenshots

Using a strong tag:

![Screenshot 2021-06-16 at 17 08 12](https://user-images.githubusercontent.com/90977/122255504-179ddc00-cec6-11eb-9dc6-91267f250810.png)

### How to test the changes in this Pull Request:

1. Edit a product and add HTML to the title e.g. `<strong>Something</strong>`
2. View the product within the All Products Block
3. Confirm the strong tag is rendered, rather than the HTML as a string



### Changelog

> Allow HTML in All Products Block Product Titles.
